src/swapnaWJun20/PrimeOperations.java

```diff
@@ -0,0 +1,53 @@
+/*Program 1: Create a class to satisfy below requirements. 
+                  a) Given number is prime or not. 
+                  b) Print total count of prime numbers between 2->100.
+                  c) Find sum of all prime numbers between 2->100
+                  d) Find avg of prime numbers between 2->100*/
+
+package swapnaWJun20;
+
+public class PrimeOperations {
+
+	void totalCount(int a, int b) {
```

Comment:
Method name is not expected like this... findPrimeNumberInGivenRange(first,last)

Reply:
@KrishnaKTechnocredits Changes done as per the comments.